zha/zigbee/cluster_handlers/lighting.py

```diff
@@ -65,7 +63,7 @@ class ColorClusterHandler(ClusterHandler):
         Color.AttributeDefs.options.name: True,
     }
 
-    @cached_property
```

Comment:
Is color_capabilities changing?

Reply:
Ah, I think this was a hack to get the unit test to pass. Fixed, thanks!